ci/checks/check_only_name_tag_snippets.py

```diff
@@ -0,0 +1,127 @@
+"""
+Purpose: To ensure that no docs snippets use the old "Mark Down" style of including Python code snippets.
+
+The old "Mark Down" style of including Python code snippets has the following form:
+
+```python Python code
+import great_expectations as gx
+
+context = gx.get_context()
+```
+
+It can also be expressed in the following form:
+
+```python title="Python code"
+import great_expectations as gx
+
+context = gx.get_context()
+```
+
+However, the new style of including Python code snippets refers to the Python module, containing the test, as follows:
+
+```python name="tests/integration/docusaurus/general_directory/specic_directory/how_to_do_my_operation.py get_context"
+```
+
+whereby "tests/integration/docusaurus/general_directory/specic_directory/how_to_do_my_operation.py get_context", which
+is the Python module, containing the integration test in the present example, would contain the following tagged code:
+
+# Python
+# <snippet name="tests/integration/docusaurus/general_directory/specic_directory/how_to_do_my_operation.py get_context">
+import great_expectations as gx
+
+context = gx.get_context()
+# </snippet>
+
+Adherence to this pattern is assertained by the present checker module.
+"""
+
+import pathlib
+import re
+import shutil
+import subprocess
+import sys
+from typing import List
+
+ITEMS_IGNORED_FROM_NAME_TAG_SNIPPET_CHECKER = {
+    "docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/how_to_connect_to_postgresql_data.md",
```

Comment:
These lines should be deleted once the implementation is completed and merged.  This effort is Work-In-Progress and is being tracked via the JIRA ticket indicated in the description.